config/src/config/execution_config.rs

```diff
@@ -44,6 +45,8 @@ impl Default for ExecutionConfig {
             genesis_file_location: PathBuf::new(),
             // Default value of 30 seconds for the network timeout.
             network_timeout_ms: 30_000,
+            // Sequential execution by default.
+            concurrency_level: 1,
```

Comment:
wow <3

Reply:
much better than env variables for sure :)